tools/pipelines/templates/build-npm-package.yml

```diff
@@ -277,6 +281,33 @@ stages:
               tagName: ${{ parameters.tagName }}
               interdependencyRange: ${{ parameters.interdependencyRange }}
 
+        # Generate rollups
+        - ${{ if or(eq(parameters.releaseBuildOverride, 'prerelease - alpha'), eq(parameters.releaseBuildOverride, 'prerelease - beta')) }}:
+          - task: Npm@1
+            displayName: npm run ${{ parameters.taskGenerateRollups }}
+            inputs: 
+              command: 'custom'
+              workingDir: ${{ parameters.buildDirectory }}
+              customCommand: 'run ${{ parameters.taskGenerateRollups }}'
+        
+        # Update package.json types to alpha
+        - ${{ if (eq(parameters.releaseBuildOverride, 'prerelease - alpha') }}:
+          - task: Npm@1
+            displayName: Set package.json types to alpha
```

Comment:
Once you rename the variable values to be alpha/beta, then you can have a single step for both.